gulpfile-extensions-example.js

```diff
@@ -27,6 +33,65 @@ module.exports = taskMethods => {
 		});
 	};
 
+	/**
+	 * overwrites the launchE2O function in gulpfile.js
+	 */
+	taskMethods.launchE2O = done => {
+		let electronProcess = null;
+		let manifest = taskMethods.startupConfig[process.env.NODE_ENV].serverConfig;
+		process.env.ELECTRON_DEV = true;
+		ON_DEATH((signal, err) => {
+
+			if (electronProcess) electronProcess.kill();
+
+			exec("taskkill /F /IM electron.* /T", (err, stdout, stderr) => {
+				// Only write the error to console if there is one and it is something other than process not found.
+				if (err && err !== 'The process "electron.*" not found.') {
+					console.error(errorOutColor(err));
+				}
+
+				if (watchClose) watchClose();
```

Comment:
watchClose is throwing a not defined exception when I ctrl+c to kill finsemble. I'm not sure about regular shutdown because of the existing childprocess error, but this variable is not defined in this file.